### PR TITLE
Add Ublox ZED-F9P-15B

### PR DESCRIPTION
**Pull request by AlexKlimaj**
This PR adds support for the Ublox ZED_F9P-15B. Requires https://github.com/PX4/PX4-GPSDrivers/pull/150

### CODE REVIEW

File: src/drivers/gps/params.c [hunk 1]
```diff
@@ -277,14 +279,16 @@ PARAM_DEFINE_INT32(GPS_1_GNSS, 0);
  * 2 : Use Galileo
  * 3 : Use BeiDou
  * 4 : Use GLONASS
+ * 5 : Use NAVIC
  *
  * @min 0
- * @max 31
+ * @max 63
```
Comment (julianoes) [inline src/drivers/gps/params.c:285]: Nice, didn't know about this one.

**Review by julianoes — DISMISSED** [review]

**Comment by MaEtUgR** [comment]
Let's update the hash to the merged driver and bring it in.

**Comment by AlexKlimaj** [comment]
Updated GPS submodule.

**Comment by AlexKlimaj** [comment]
@dagar rebased on main and pushed.

**Comment by julianoes** [comment]
Rebased, fixed up and force pushed.

**Review by julianoes — APPROVED** [review]

**Pull request merged**